cirq/sim/wave_function_test.py

```diff
@@ -11,14 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Tests for wavefunction.py"""
+"""Tests for wave_function.py"""
 
 import itertools
 import pytest
 
 import numpy as np
 
 import cirq
+from cirq import sim
```

Comment:
Just `import cirq` in tests

Reply:
The method I moved was only exposed at cirq.sim because I don't think it sould be part of the public interface.  But now it is moved into step result, and has a better signature.